ext/nnstreamer/tensor_filter/tensor_filter_edgetpu.cc

```diff
@@ -144,10 +173,71 @@ tensor_filter_subplugin & edgetpu_subplugin::getEmptyInstance ()
   return *(new edgetpu_subplugin());
 }
 
+/**
+ * @brief Internal helper to get lowercase string of the given one.
+ * @param s The given std:string value
+ * @return A lowercase version of s
+ */
+std::string edgetpu_subplugin::str_tolower(std::string s) {
+  std::transform(s.begin(), s.end(), s.begin(),
+      [](unsigned char c){ return std::tolower(c); });
+
+  return s;
+}
+
+/**
+ * @brief Internal method to parse custom properties
+ * @param custom_prop The given c_str value of the 'custom' property
+ * @return A edgetpu_subplugin_device_type value for the given 'device_type'
+ *        if the value of the 'custom' property. Otherwise,
+ *        edgetpu_subplugin_device_type::USB, which is default, is returned.
+ */
+edgetpu_subplugin_device_type edgetpu_subplugin::parse_custom_prop (
+    const char *custom_prop)
+{
+  if ((!custom_prop) || (strlen (custom_prop) == 0))
+    return edgetpu_subplugin_device_type::USB;
```

Comment:
minor comment - how about declaring a default device type which can be used here as well as at the constructor of `edgetpu_subplugin`.

Reply:
@kparichay I guess I do not fully understand your comment, but an enum value indicating a DEFAULT device type is added and the constructor and this function are modified to use this enum as well.

However, IMHO, USB is more intuitive than DEFAULT to me.

Reply:
My only concern is that if the default device is updated in future, it would require changes in many locations.
If USB looks more intuitive, then please keep the USB.

Reply:
@kparichay I have replaced USB, which is used as a default device, with DEFAULT.